libs/ledgerjs/packages/cryptoassets/src/tokens.ts

```diff
@@ -38,6 +39,37 @@ type TokensListOptions = {
 const defaultTokenListOptions: TokensListOptions = {
   withDelisted: false,
 };
+export function createTokenHash(token: TokenCurrency): string {
+  return token
+    ? `${token.id}${token.contractAddress}${token.delisted}${token.disableCountervalue}${token.ticker}${token.ledgerSignature}`
```

Comment:
Not using `countervalueTicker` means a dyn cal update of the ticker won't be applied, right?
What was the reason for removing it from the hash?

Reply:
Nvm, recent rework removed it from the type https://github.com/LedgerHQ/ledger-live/commit/4a1454beb3f86405ba2686e07879c12a7d35ad8e